helm/templates/deploy.yaml

```diff
@@ -63,11 +63,11 @@ spec:
         image: {{ .Values.image.psql.repository }}:{{ .Values.image.psql.tag }}
         resources:
           limits:
-            cpu: 50m
-            memory: 64Mi
+            cpu: {{ .Values.spilo.resources.limits.cpu }}
+            memory: {{ .Values.spilo.resources.limits.memory }}
```

Comment:
You don't need/want to change any of this. The containers running psql need way less resources than the ones running the actual database

Reply:
If I find that the init container run painfully slow and I have resources, can I bring these numbers up a bit?